src/ChatWindow.test.js

```diff
@@ -21,27 +21,38 @@ it("renders ChatWindow without crashing", () => {
   mount(<ChatWindow />);
 });
 
-it("that it renders correctly (snapshot testing)", () => {
+it("renders correctly (snapshot testing)", () => {
+  // Set height of the ChatFeed container.
+  Element.prototype.getBoundingClientRect = jest.fn(() => {
+    return {
+      height: CHATFEED_CONTAINER_HEIGHT
+    };
+  });
+
   const wrapper = mount(<ChatWindow />);
   expect(wrapper).toMatchSnapshot();
-});
 
-it("that height of container is passed to ChatFeed", () => {
-  const wrapper = mount(<ChatWindow />);
-  const chatFeed = wrapper.find(ChatFeed);
+  wrapper.unmount();
+});
 
+it("passes height of container to ChatFeed component", () => {
   // Set height of the ChatFeed container.
   Element.prototype.getBoundingClientRect = jest.fn(() => {
     return {
       height: CHATFEED_CONTAINER_HEIGHT
     };
   });
 
+  const wrapper = mount(<ChatWindow />);
+  const chatFeed = wrapper.find(ChatFeed);
+
   expect(chatFeed.length).toBe(1);
   expect(chatFeed.prop("maxHeight")).toBe(CHATFEED_CONTAINER_HEIGHT);
+
+  wrapper.unmount();
```

Comment:
This can be moved to afterEach()  if you have wrapper initialized in beforeEach().

Even though there are only 2-3 tests, it is still a good practice to mount and unmount in beforeEach and afterEach classes.

Reply:
Unlike 'shallow', 'mount' is an expensive operation and so we probably don't want to mount it before each test if we don't really need it. So, I will leave it as is.